src/main/java/org/broadinstitute/hellbender/utils/bigquery/TableReference.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.hellbender.utils.bigquery;
+
+import java.util.List;
+
+public class TableReference {
+    public final String tableProject;
+    public final String tableDataset;
+    public final String tableName;
+    public final List<String> fields;
```

Comment:
If `fields` is going to be public like this, you should wrap it in an `ImmutableList` or similar.